staging/src/k8s.io/code-generator/cmd/validation-gen/validation.go

```diff
@@ -910,10 +910,16 @@ func (g *genValidations) emitValidationForChild(c *generator.Context, thisChild
 			// If the node is nil, this must be a type in a package we are not
 			// handling - it's effectively opaque to us.
 			if fld.node == nil {
-				targs := generator.Args{
-					"type": fld.childType,
+				if fld.fieldValidations.SkipUnimported {
```

Comment:
What about all the other places - list values, maps, etc?

Reply:
> What about all the other places - list values, maps, etc?

This is turning out to be more challenging to support.  I'm working through the logic, but it's twisty.